dipy/viz/horizon/app.py

```diff
@@ -92,7 +92,12 @@ def __init__(
         roi_colors=(1, 0, 0),
         surface_colors=((1, 0, 0),),
     ):
-        """Interactive medical visualization - Invert the Horizon!
+        """""Orchestrates interactive medical visualization for multimodal data.
```

Comment:
- no `s` for `Orchestrates`.  Imperative mood
- Also, why 5 double quotes instead of 3?